evap/staff/views.py

```diff
@@ -2289,11 +2289,11 @@ class UserMergeSelectionView(FormView):
     form_class = UserMergeSelectionForm
     template_name = "staff_user_merge_selection.html"
 
-    def get_success_url(self):
+    def form_valid(self, form):
         return redirect(
             "staff:user_merge",
-            self.form.cleaned_data["main_user"].id,
-            self.form.cleaned_data["other_user"].id,
+            form.cleaned_data["main_user"].id,
+            form.cleaned_data["other_user"].id,
```

Comment:
@niklasmohrin from what I currently understand, `self.form` was not defined here (at least I can not find why it should be defined in django's docs), and `get_success_url` should probably return an url, not a HttpResponse (as redirect does). Is there a chance that we messed this up with other views as well?

(The code form this PR seems to be ok to me according to the docs)

Reply:
Oh no I think we were literally talking about this code and agreed that it is trivial enough to not test or something 😂: https://github.com/e-valuation/EvaP/pull/1964#discussion_r1265804150

Reply:
But in all honesty, we should probably go over the generic views PR again and maybe add types or so? Its mostly just classes with variables, those are fine, but the rest we should check to be sure

Reply:
Created #2055 for this. Resolving here.